azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/header/InfoHeaderView.kt

```diff
@@ -57,18 +58,35 @@ internal class InfoHeaderView : ConstraintLayout {
         }
 
         viewLifecycleOwner.lifecycleScope.launch {
+            val appLocalizationProvider = infoHeaderViewModel.getApplicationLocalizationProvider()
             infoHeaderViewModel.getNumberOfParticipantsFlow().collect {
                 when (it) {
-                    0 -> participantNumberText.setText(R.string.azure_communication_ui_call_waiting_for_others_to_join)
-                    1 -> participantNumberText.setText(R.string.azure_communication_ui_call_call_with_1_person)
+                    0 -> participantNumberText.text = appLocalizationProvider.getLocalizedString(
```

Comment:
can we extract a string variable and assign it in the when() {} 
then set it to the text:
participantNumberText.text = appLocalizationProvider.getLocalizedString(
                        context,
                        key
                    )